CricketGame.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Problem : Create a cricket game using java.
+ * Owner Name : Arjun Gautam
+ * Date : 17/09/24
+ */
+package Assignment_5;
+import java.util.Scanner;
+
+/*
+ * The Team class stores a team's name (teamName), a 2D array of player names and roles (players),
+ * and tracks the number of players (playerCount), with the addPlayer() method adding players if the team isn't full.
+ */
+class Team {
+    String teamName;
+    String[][] players = new String[11][2]; 
+    int playerCount = 0; 
+
+    Team(String teamName) {
+        this.teamName = teamName;
+    }
+    void addPlayer(String playerName, String playerRole) {
+        if (playerCount < 11) {
+            players[playerCount][0] = playerName;
+            players[playerCount][1] = playerRole;
+            playerCount++;
+        } else {
+            System.out.println("Team is full. Cannot add more players.");
+        }
+    }
+}
+
+public class CricketGame {
+
+    private static Team[] teams = new Team[10]; 
+    private static int teamCount = 0; 
+
+/*
+ * The viewTeams() method checks if teams exist, prints each team's name, and for each team, 
+ * it either shows "No players" if playerCount is 0 or lists all players and their roles.
+ */
+    public static void viewTeams() {
+        if (teamCount == 0) {
+            System.out.println("No teams have been created yet.");
+        } else {

```

Comment:
Make sure that if-else conditions show thereself in a clean way (Its look messy)